js-modules/prettify.js

```diff
@@ -59,6 +59,8 @@
 
 include("defs.js");
 
+var HACK_TO_FIX_JS_INCLUDE_PL;
```

Comment:
no, I got rid of it later, along with the perl scripts and the entire makefile.

(some of these fixes were made before I started working on porting to a grunt solution)